src/mega-plusplus.coffee

```diff
@@ -111,6 +103,12 @@ module.exports = (robot) ->
         from:      from
       }
 
+  # sweet regex bro
+  robot.hear plusPlusRegex, (msg) ->
+    msg.match.forEach (match, index, array) ->
+      [_, name, operator, reason] = match.match(new RegExp(plusPlusRegex, 'i'))
```

Comment:
Whoops, I think this actually overwrites the `_` declared at the top of the file 😓